Analysis/src/EventInfo.cpp

```diff
@@ -386,6 +391,22 @@ const kin_fit::FitResults& EventInfoBase::GetKinFitResults()
     return *kinfit_results;
 }
 
+const sv_fit_ana::FitResults& EventInfoBase::GetSVFitResults()
+{
+    Lock lock(*mutex);
+    if(!svfit_results){
```

Comment:
wrong logic. Like this `svfit_results` will never be created. Implementation should be similar to `GetKinFitResults`.

Reply:
please implement this comment